scripts/create_test_subset.py

```diff
@@ -165,8 +164,12 @@
     query ($project: String!) {
         project (name: $project) {
             participants {
+                externalIds
```

Comment:
we query for participants in the target project twice - one requires all these attributes, one only requires externalIds and id. This updates the one query to return all the data to both, negating the need for similar-but-different GraphQL queries